### PR TITLE
Dragons End Buffs and Mechanics

### PR DESCRIPTION
**Pull request by sreilly64**
This PR adds various buffs and mechanics specific to the Soo-Won fight in Dragon's End.
- Added buffs Jade Tech Overcharges, Soul Reunited, Hardened Scales, and Enhancement (+5% damage vs. Dragonvoid) to buffs list
- Added debuffs Void Corruption and Void Chaos to buff list
- Added Soul Reunited to Damage Modifiers
- Added Soo-won's attacks Tsunami Slam, Acid, Claw Slap, Tail Slap, Bite, Tidal Waves (half and full platform), Bubbles, and Whirlpools as tracked mechanics
- Added Side Swap, Tail spawns, green circle/wisp form phase, and failed wisps as tracked mechanics
- Added successful breakbars as a tracked mechanic

Things not included that I think would be useful and will look into:
- Implement failed CC as a tracked mechanic
- Implement tracking for Dragon's End Contributor buffs in a concise way

### CODE REVIEW

File: GW2EIEvtcParser/EncounterLogic/OpenWorld/SooWon.cs [hunk 1]
```diff
@@ -12,6 +12,25 @@ internal class SooWon : FightLogic
     {
         public SooWon(int triggerID) : base(triggerID)
         {
+            MechanicList.AddRange(new List<Mechanic>
+            { 
+            new HitOnPlayerMechanic(67551, "Tsunami Slam", new MechanicPlotlySetting("triangle-down",Colors.DarkRed), "Slam", "Soo-Won slams the ground in front of her creating a circular tsunami", "Tsunami Slam", 0),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EncounterLogic/OpenWorld/SooWon.cs:17]: All skill ids should be in SkillIDs.cs

**Comment by EliphasNUIT** [comment]
For Dragon's End Contributor from what I see the ids are:
- 63928 (20 %)
- 64250 (3%)
- 64623 (8%)
- 64924 (4%)
- 65384 (1%)
- 66185 (2%)
- 66589 (5%)
- 67046 (9%)
- 67352 (6%)
- 67549 (7%)
In this case, outgoing damage also means "strike, condition and life leech".

**Comment by EliphasNUIT** [comment]
Looks great! I've just fixed the stack type for Jade Tech buffs from Stacking to Queue.

Thanks!

**Comment by sreilly64** [comment]
I totally missed the BuffStackType.Queue was even a thing. Thanks for fixing that and for the feedback!

**Pull request merged**